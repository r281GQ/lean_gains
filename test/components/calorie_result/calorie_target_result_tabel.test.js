import React from 'react';
import { shallow, render, mount } from 'enzyme';
import CalorieTargetResultTable from './../../../src/components/calorie_result/calorie_target_result_table';

describe('calorie_target_result_sum_label', () => {
  it('should render label with given properties', () => {
    const component = shallow(
      <CalorieTargetResultTable
        label="rest"
        calorieTarget={{ rest: { calorie: 1500, protein: 100, fat: 50, carbohydrate: 100 } }}
      />
    );
    expect(component).toMatchSnapshot();
  });
});
