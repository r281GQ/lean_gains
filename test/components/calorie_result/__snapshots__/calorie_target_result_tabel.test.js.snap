// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calorie_target_result_sum_label should render label with given properties 1`] = `
<Table
  allRowsSelected={false}
  fixedFooter={true}
  fixedHeader={true}
  height="inherit"
  multiSelectable={false}
  selectable={true}
>
  <TableHeader
    adjustForCheckbox={true}
    displaySelectAll={false}
    enableSelectAll={true}
    selectAllSelected={false}
  >
    <TableRow
      displayBorder={true}
      hoverable={false}
      hovered={false}
      selectable={false}
      striped={false}
    >
      <TableHeaderColumn />
      <TableHeaderColumn>
        Protein
      </TableHeaderColumn>
      <TableHeaderColumn>
        Carbohydrate
      </TableHeaderColumn>
      <TableHeaderColumn>
        Fat
      </TableHeaderColumn>
    </TableRow>
  </TableHeader>
  <TableBody
    allRowsSelected={false}
    deselectOnClickaway={true}
    displayRowCheckbox={false}
    multiSelectable={false}
    preScanRows={true}
    selectable={true}
    style={Object {}}
  >
    <TableRow
      displayBorder={true}
      hoverable={false}
      hovered={false}
      selectable={false}
      striped={false}
    >
      <TableRowColumn
        hoverable={false}
      >
         Calories
      </TableRowColumn>
      <TableRowColumn
        hoverable={false}
      >
        <CalorieTargetLabel
          value={400}
        />
      </TableRowColumn>
      <TableRowColumn
        hoverable={false}
      >
        <CalorieTargetLabel
          value={400}
        />
      </TableRowColumn>
      <TableRowColumn
        hoverable={false}
      >
        <CalorieTargetLabel
          value={450}
        />
      </TableRowColumn>
    </TableRow>
    <TableRow
      displayBorder={true}
      hoverable={false}
      hovered={false}
      selectable={false}
      striped={false}
    >
      <TableRowColumn
        hoverable={false}
      >
         Grams
      </TableRowColumn>
      <TableRowColumn
        hoverable={false}
      >
        <CalorieTargetLabel
          value={100}
        />
      </TableRowColumn>
      <TableRowColumn
        hoverable={false}
      >
        <CalorieTargetLabel
          value={100}
        />
      </TableRowColumn>
      <TableRowColumn
        hoverable={false}
      >
        <CalorieTargetLabel
          value={50}
        />
      </TableRowColumn>
    </TableRow>
  </TableBody>
</Table>
`;
